pkg/azuredevops/pipelines/templatesParams.go

```diff
@@ -149,8 +149,8 @@ func (p OCIImageBuilderTemplateParams) Validate() error {
 	if jobType, ok = p["JobType"]; !ok {
 		return ErrRequiredParamNotSet("JobType")
 	}
-	if jobType != "presubmit" && jobType != "postsubmit" && jobType != "on-demand" {
-		return fmt.Errorf("JobType must be either presubmit, postsubmit or on-demand, got: %s", jobType)
+	if jobType != "presubmit" && jobType != "postsubmit" && jobType != "workflow_dispatch" {
+		return fmt.Errorf("JobType must be either presubmit, postsubmit or workflow_dispatch, got: %s", jobType)
```

Comment:
We should consider checking if specific string is in slice. For more possible jobTypes it would be no more readable.